runtime/src/jsMain/kotlin/pbandk/protobufjs/Util.kt

```diff
@@ -28,13 +30,7 @@ object Util {
 
     fun bytesToBase64(bytes: ByteArray): String = bytes.asUint8Array().let { util.base64.encode(it, 0, it.length) }
 
-    fun timestampToString(ts: Timestamp): String =
-        Date((ts.seconds * 1000) + (ts.nanos / 1_000_000)).toISOString()
+    fun timestampToString(ts: Timestamp): String = formatTime(ts.seconds, ts.nanos)
```

Comment:
Since we have all of the platforms now using the methods from `TimeUtil`, let's add `timestampToString()` and `stringToTimestamp()` to the `AbstractUtil` class and then have `Util` here and `Util` in `jvmMain` both inherit from `AbstractUtil`. You can see an example of this in the `Util` class in `nativeMain`.